static/src/js/components/SpatialDisplay/SpatialDisplay.js

```diff
@@ -345,6 +360,26 @@ class SpatialDisplay extends Component {
     e.preventDefault()
   }
 
+  /**
+   * Validates a Lat/Lng is limited to 5 decimal points
+   * @param {*} latLng
+   */
+  isValidDecimalLatLng(latLng) {
```

Comment:
Methods should be prefixed with `is` when they return a boolean, this returns an array. I'd change the name of the method or ensure this method returns a boolean `return latLng.match(regex).length > 0` or something similar.